data/react-entrypoint-generator.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * This is a script that generates static React class files for two 
+ * different scenarios:
+ *
+ * 	1. A state page that displays all the congressmen in that
+ * 	   state.
+ * 	2. A district page that displays all the congressmen in that
+ * 		 district (typically based on full address).
+ */
+const React = require('react');
+const fs = require('fs');
+const forEach = require('lodash/forEach');
+const forOwn = require('lodash/forOwn');
+const syncExec = require('sync-exec');
+
+const govData = JSON.parse(
+	fs.readFileSync('./data/people.json', 'utf8')
+);
+
+// Loop through the government data and start to sort them by
+// state and district.
+let congressByStateAndDistrict = {};
+
+forEach(govData, function(person) {
+	if (!person.current) {
+		console.log(`${person.person.name} is not a current worker in government.`);
+		return;
+	}
+
+	// If this isn't a congressman with a district, it is likely a Senator.
+	// We don't want to deal with Senator data just yet.
+	if (!person.district) {
+		console.log(`${person.person.name} is not a Congressman.`);
```

Comment:
`console.error` is a bit more descriptive here, and some terminals will style it differently than the regular `log` messages.